### PR TITLE
Bump matrix-react-test-utils for React 16 compatibility

### PR DESCRIPTION
**Pull request by t3chguy**
Signed-off-by: Michael Telatynski <7t3chguy@gmail.com>

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6397,7 +6394,7 @@ react-dom@^16.4.2:
   version "2.1.5"
   resolved "https://codeload.github.com/matrix-org/react-gemini-scrollbar/tar.gz/f64452388011d37d8a4427ba769153c30700ab8c"
   dependencies:
-    gemini-scrollbar matrix-org/gemini-scrollbar#91e1e566
+    gemini-scrollbar matrix-org/gemini-scrollbar#b302279
```
Comment (jryans) [inline yarn.lock]: Hmm, something seems wrong here... matrix-org/react-gemini-scrollbar#f644523 depends on matrix-org/gemini-scrollbar#91e1e566, so I don't think this part of the lock file should have changed... Maybe try deleting this single block form `yarn.lock` and re-running `yarn install`?

Comment (t3chguy) [reply yarn.lock]: Indeed this is funky

Comment (t3chguy) [reply yarn.lock]: that seems to have worked

Comment (jryans) [reply yarn.lock]: That tends to be the best procedure to get `yarn` to do sensible things when tweaking a specific package, even though it feels like it should have to be that way... 😅

**Pull request merged**